main.cpp

```diff
@@ -98,10 +98,20 @@ struct Laundromat
     float makeChange(float price, float amPaid);
 };
 
-void Laundromat::washClothes()
+Laundromat::Laundromat()
 {
+    numWashers = 0;
+    numDryers = 0;
+    amOfWaterUsedDaily = 0;
+    amOfEnergyUsedDaily = 0;
+    amOfMoneyMadeDaily = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise